data/remnant outfits.txt

```diff
@@ -359,11 +359,31 @@ outfit "Salvage Scanner"
 
 # Engines:
 
+outfit "Anvil-Class Engine"
+	category "Engines"
+	licenses
+		Remnant
+	"cost" 290000
+	thumbnail "outfit/tiny remnant engine"
+	"mass" 26
+	"outfit space" -26
+	"engine capacity" -26
+	"thrust" 13.5
+	"thrusting energy" 2.65
+	"thrusting heat" 2.5
+	"turn" 336.0
+	"turning energy" 1.32
+	"turning heat" 1.4
```

Comment:
Again, the turning energy and heat shouldn't match the Crucible-Class if the turn is lesser. I'd suggest 1.05 energy and 1.1 heat.

Reply:
OK. Changed